docs/Reference/CLI/CLI-Syntax.md

```diff
@@ -314,8 +314,9 @@ TEKU_METRICS_CATEGORIES=BEACON,JVM,PROCESS
 metrics-categories: ["BEACON", "JVM", "PROCESS"]
 ```
 
-Categories for which to track metrics. Options are `BEACON`, `JVM`, `LIBP2P`, `NETWORK`, and
-`PROCESS`. All categories are enabled by default.
+Categories for which to track metrics. Options are `BLOCKCHAIN`, `ETHEREUM`, `EXECUTORS`, 
+`NETWORK`, `PEERS`, `PERMISSIONING`, `PRUNER`, `RPC`, `SYNCHRONIZER`, `TRANSACTION_POOL`, `JVM`,
+and `PROCESS`. All categories are enabled by default.
```

Comment:
I'm not sure where a bunch of these categories came from. I've got:
```
--metrics-categories[=<METRICS_CATEGORY>[,<METRICS_CATEGORY>...]...]
                             Metric categories to enable
                               Default: BEACON,LIBP2P,NETWORK,EVENTBUS,JVM,
                               PROCESS
```

and all the default ones are basically all the possible categories...

Reply:
these are besu categories?

Reply:
those are besu categories?